src/SourceGenerators/Uno.UI.SourceGenerators/Content/Uno.UI.SourceGenerators.props

```diff
@@ -63,6 +63,7 @@
 		<CompilerVisibleProperty Include="UnoXamlResourcesTrimming"/>
 		<CompilerVisibleProperty Include="BaseIntermediateOutputPath"/>
 		<CompilerVisibleProperty Include="AssemblyName"/>
+		<CompilerVisibleProperty Include="SolutionFileName"/>
```

Comment:
@jeromelaban This part is under `Condition="'$(UnoUIUseRoslynSourceGenerators)'=='true'"`

For the failing files, `UnoUIUseRoslynSourceGenerators` is false. Meaning that `SolutionFileName` isn't accessible.

Should I remove the condition? or there is something wrong with the value of `UnoUIUseRoslynSourceGenerators`?

Reply:
This specific property is only used when using Roslyn hosting, not when using Uno.SourceGenerationTasks. The `SolutionFileName` should still be available. 

If not, we'll need to create another property that is unique to the uno solution, e.g. adding it to the `Directory.Build.props` file and add it to the `CompilerVisibleProperty` list instead.